draft-ietf-rats-architecture.md

```diff
@@ -1424,30 +1425,57 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
 
 ## Example 3: Handle-based Passport Model Example
 
-Handles are a third option to establish time-keeping next to nonces or timestamps.
-Handles are opaque data intended to be available to all RATS roles that interact with each other, such as the Attester or Verifier, in specified intervals.
-To enable this availability, handles are distributed centrally by the Handle Distributor role over the network.
-As any other role, the Handle Distributor role can be taken on by a dedicated entity or collapsed with other roles, such as a Verifier.
-The use of handles can compensate for a lack of clocks or other sources of time on entities taking on RATS roles.
-The only entity that requires access to a source of time is the entity taking on the role of Handle Distributor.
+A third option to establish the epoch of Evidence or Attestation Result
+generation - next to nonces and timestamps ({{freshness}}) - is by using
+"handles".
```

Comment:
```suggestion
The following example illustrates a hypothetical Passport Model
solution that uses handles instead of timestamps or nonces.
```
For consistency with the start of examples 1 and 2